_pages/en_US/finalizing-setup.txt

```diff
@@ -139,6 +139,11 @@ If, before following this guide, you already had an EmuNAND setup and would like
 1. Press (B) to return to the main menu
 1. Select "Exit"
 1. Press (A) to relock write permissions if prompted
+1. Navigate to `[S:] SYSNAND VIRTUAL`
+1. Press (A) on `essential.exefs` to select it
+1. Select "Copy to 0:/gm9/out"
+  + If prompted that "Destination already exists", press (A) on "Overwrite file(s)"
```

Comment:
Maybe use `If you see "Destination already exists"`

Reply:
Amended.